src/app/components/home/home.component.ts

```diff
@@ -1,12 +1,14 @@
-import { Component, EventEmitter, Output } from '@angular/core';
+import { Component, EventEmitter, Output, ViewEncapsulation } from '@angular/core';
 
 @Component({
   selector: 'app-home',
   standalone: true,
   imports: [],
   templateUrl: './home.component.html',
   styleUrl: './home.component.css',
+  encapsulation: ViewEncapsulation.None,
```

Comment:
Why have you turned view encapsulation off?

Reply:
I turned it off so that the styles would be applied to score component ( for dark mode toggle)